dbt/adapters/databricks/impl.py

```diff
@@ -107,6 +108,10 @@ class DatabricksAdapter(SparkAdapter):
 
     AdapterSpecificConfigs = DatabricksConfig
 
+    _capabilities = CapabilityDict(
+        {Capability.TableLastModifiedMetadata: CapabilitySupport(support=Support.Full)}
+    )
+
     @available.parse(lambda *a, **k: 0)
```

Comment:
Is the first time our dialect has needed to use `_capabilities`? Or had we just inherited the `_capabilities` from dbt-spark? Not a blocker for this PR but I wonder if more capabilities should be reflected explicitly here.

Reply:
This is new in 1.7